examples/dgmulti_2d/elixir_navierstokes_convergence.jl

```diff
@@ -4,14 +4,14 @@ using Trixi
 ###############################################################################
 # semidiscretization of the ideal compressible Navier-Stokes equations
 
-reynolds_number() = 100
 prandtl_number() = 0.72
+viscosity() = 0.01
```

Comment:
I am open to changing the name to `kinematic_viscosity` but I need to check my fluids textbook to remind myself if this is the proper name. My personal preference would to not use the Greek letter nu, I find it too easily confused with `v` which is the velocity. We can name the vairable `nu` if we want though because then there is no confusion.

Reply:
Sorry, I think my statement was ambiguous. I decidedly do *not* want to use the Greek letters - I just wanted to ask which of the viscosities we should use :-). Using `nu` or `mu` would be ok for me too, although I also like the long form `viscosity`, since it's more descriptive.

Reply:
I updated the name to be `dynamic_viscosity` in the elixirs.

Reply:
Great, thanks!